### PR TITLE
WV-652 Challenge About [Team Review]

### PR DESCRIPTION
**Pull request by Succorro**
### What github.com/wevote/WebApp/issues does this fix?
- Challenge About section updates
### Changes included this pull request?
- Fixed SVG styling: the svg was not copied correctly resulting in improper rendering on the page
- refactored code to match figma screenshot
- created more styled components for easier readability 
- minor styling additions for font weight 
- wired up data from challenge store

### CODE REVIEW

File: src/js/common/components/Challenge/ChallengeAbout.jsx [hunk 1]
```diff
@@ -59,29 +57,6 @@ function ChallengeAbout ({ challengeWeVoteId, showDaysLeft }) {
   }, [challengeWeVoteId]);
 
   // Variables to hold dummy data
-  const challengeDates = (
```
Comment (DaleMcGrew) [inline src/js/common/components/Challenge/ChallengeAbout.jsx]: Hi @Succorro could you please roll back the challengeDates code to how it was?

File: src/js/common/components/Challenge/ChallengeAbout.jsx [hunk 2]
```diff
@@ -138,30 +148,7 @@ function ChallengeAbout ({ challengeWeVoteId, showDaysLeft }) {
             </FlexDivLeft>
           </Suspense>
         </CardForListRow>
-        {showStartedBy && (
```
Comment (DaleMcGrew) [inline src/js/common/components/Challenge/ChallengeAbout.jsx:141]: Please maintain the same place in the code for your new challengeOwners changes, as it was prior to your changes (i.e, the top-to-bottom order). Please support the "showStartedBy" variable -- there are some modes where we won't show this started by section.

**Review by DaleMcGrew — CHANGES_REQUESTED** [review]
Thank you @Succorro for these changes. I have the following requests for changes before I approve this pull request.
I hope you are having a good weekend.

**Comment by Succorro** [comment]
Let me know if there are any more changes needed @DaleMcGrew  :thumbsup:

**Comment by DaleMcGrew** [comment]
Thank you @Succorro! 👍

**Pull request merged**